plugins/GSdx/GSDevice11.cpp

```diff
@@ -768,7 +768,7 @@ void GSDevice11::InitExternalFX()
 			if (fshader.good())
 			{
 				shader << fshader.rdbuf();
-				CompileShader(shader.str().c_str(), shader.str().length(), shader_name.c_str(), "ps_main", NULL, &m_shaderfx.ps);
+				CompileShader(shader.str().c_str(), shader.str().length(), shader_name.c_str(), D3D_COMPILE_STANDARD_FILE_INCLUDE, "ps_main", nullptr, &m_shaderfx.ps);
```

Comment:
I see this is again probably only me bothering since I use the good old 7.1 SDK + DX SDK (just like in #1237)...
But I get a `C2065: undeclared identifier` error here. 

Should I start a fork just for this change now?